mkosi/distributions/__init__.py

```diff
@@ -29,3 +32,8 @@ def remove_packages(cls, state: "MkosiState", remove: list[str]) -> None:
     @classmethod
     def filesystem(cls) -> str:
         raise NotImplementedError
+
+
+def cache_params(state: "MkosiState") -> list[PathString]:
```

Comment:
This is quite specific to using bubblewrap, no? I guess that would be better places in `run.py`.